spec/imsc-hrm.html

```diff
@@ -615,28 +624,28 @@ <h2>Paint Regions</h2>
     <section id='paint-images'>
       <h2>Paint Images</h2>
 
-      <p>The Presentation Compositor SHALL paint into the Presentation Buffer P<sub>n</sub> all visible pixels of <a>presented
+      <p>The Presentation Compositor SHALL paint into the Front Buffer all visible pixels of <a>presented
       images</a> of <a>Intermediate Synchronic Document</a> E<sub>n</sub>.</p>
 
       <p>For each <dfn data-cite="IMSC#dfn-presented-image">presented image</dfn>, the Presentation Compositor SHALL either:</p>
 
       <ul>
         <li>if an identical image is present in Decoded Image Buffer D<sub>n</sub>, copy the image from Decoded Image Buffer
-        D<sub>n</sub> to the Presentation Buffer P<sub>n</sub> using the Image Copier; or</li>
+        D<sub>n</sub> to the Front Buffer using the Image Copier; or</li>
 
         <li>if an identical image is present in Decoded Image Buffer D<sub>n-1</sub>, i.e. an identical image was present in
         <a>Intermediate Synchronic Document</a> E<sub>n-1</sub>, copy using the Image Copier the image from Decoded Image Buffer
-        D<sub>n-1</sub> to both the Decoded Image Buffer D<sub>n</sub> and the Presentation Buffer P<sub>n</sub>; or
+        D<sub>n-1</sub> to both the Decoded Image Buffer D<sub>n</sub> and the Front Buffer; or
         </li>
 
-        <li>otherwise, decode the image using the Image Decoder the image into the Presentation Buffer P<sub>n</sub> and Decoded
+        <li>otherwise, decode the image using the Image Decoder the image into the Front Buffer and Decoded
```

Comment:
This doesn't make grammatical sense; this change hasn't affected that one way or the other, but it looks like there are two occurrences of "the image" where only one is needed.

Reply:
See https://github.com/w3c/imsc-hrm/pull/50/commits/b48a520eaae99bb12967ac56e8755e240df3353a